packages/cf-component-loading/src/Loading.js

```diff
@@ -1,8 +1,49 @@
 import React from 'react';
-import Icon from 'cf-component-icon';
+import { Icon } from 'cf-component-icon';
 
-export default class Loading extends React.Component {
+import { createComponent } from 'cf-style-container';
+
+const loadingStyles = ({ theme }) => {
+  return {
+    width: 15,
+    animationDuration: '2.5s',
+    animationIterationCount: 'infinite',
+    animationTimingFunction: 'linear',
+    '>svg': {
+      height: '100%',
+      fill: 'transparent',
+      strokeWidth: 6,
+      stroke: theme.colors.hail,
```

Comment:
You mentioned before that this will have multiple colour options. Did we scrap that idea?

Reply:
I figured we should put more thought into where folks might want to use the loading spinner before we propose other color/style options. We still want to add multiple options in the near future.